src/SkeletalTrapezoidation.cpp

```diff
@@ -381,6 +381,11 @@ SkeletalTrapezoidation::SkeletalTrapezoidation(const Polygons& polys, float tran
     init();
 }
 
+void SkeletalTrapezoidation::setBeadingStrategy(BeadingStrategy* beading_strategy)
```

Comment:
Why not in the constructor?
We could probably keep it a reference if it's done that way.
(Also, we should probably start using the new safe pointer types at some point...)

Reply:
No idea why I didn't do that... It makes way more sense as part of the constructor. Fixed that.